contracts/exchange_L1_L2/ThalesExchanger.sol

```diff
@@ -36,8 +40,10 @@ contract ThalesExchanger is IThalesExchanger, Owned, ReentrancyGuard, Pausable {
         ThalesToken = IERC20(thalesAddress);
         OpThalesToken = IERC20(opThalesAddress);
         L1Bridge = iOVM_L1ERC20Bridge(_l1BridgeAddress);
-        OpThalesToken.approve(_l1BridgeAddress, 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff);
+        OpThalesToken.approve(_l1BridgeAddress, MAX_APPROVAL);
         l2TokenAddress = _l2TokenAddress;
+        enabledThalesToOpThales = true;
```

Comment:
set this as inline initialization in lines 25 and 26

Reply:
ok